src/main/java/com/serhiiklymchuk/gmailpet/controller/ErrorController.java

```diff
@@ -0,0 +1,33 @@
+package com.serhiiklymchuk.gmailpet.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Controller
+public class ErrorController implements org.springframework.boot.web.servlet.error.ErrorController {
+
+    @GetMapping("/error")
+    @ResponseBody
+    public String showError(HttpServletRequest req){
+        Integer statusCode = (Integer) req.getAttribute("javax.servlet.error.status_code");
+        Exception exception = (Exception) req.getAttribute("javax.servlet.error.exception");
+
+        return String.format("<html>" +
```

Comment:
Use ftl template for rendering html.
This isn't the responsibility of the controller.